api/src/main/java/org/pmiops/workbench/api/UserController.java

```diff
@@ -63,11 +67,17 @@ public UserController(UserService userService) {
         .ofNullable(Sort.Direction.fromStringOrNull(sortOrder))
         .orElse(Sort.Direction.ASC);
     Sort sort = new Sort(new Sort.Order(direction, DEFAULT_SORT_FIELD));
+
+    List<User> users = userService.findUsersBySearchString(term, sort);
+
     // We want to filter out users not initialized in firecloud yet to avoid sharing with a not yet existent user.
-    List<User> users = userService.findUsersBySearchString(term, sort)
-        .stream()
-        .filter(user -> user.getFreeTierBillingProjectName() != null)
-        .collect(Collectors.toList());
+    // This is no longer relevant if we create new workspaces with new billing projects
```

Comment:
Ideally we'd still filter out users who had never signed in - as they will not be registered in Firecloud (else the FC share call will fail with some opaque error). It's a bit of an indirect check, but I'd suggest just changing this code (and don't fork) to check for firstSignInTime != null